Tests/Foundation/Tests/PolymorphicCodableTests.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  PolymorphicCodableTests.swift
+//  TestFoundation
+//
+//  Created by Fabian Fett on 02.03.21.
+//  Copyright © 2021 Swift. All rights reserved.
+//
```

Comment:
Needs a better header? Or maybe file is redundant?

Reply:
urgh. this file was not intended to be commited.